### PR TITLE
feat: cart screen added

### PR DESCRIPTION
**Pull request by ParthParolekar**
Added the cart screen and also made it responsive.
live: https://deploy-preview-5--suspicious-rosalind-5520a4.netlify.app/components/cart/cart

### CODE REVIEW

File: Components/cart/cart.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+                  ><span class="selling-price">₹1,600</span>
+                </div>
+              </div>
+              <div class="item-quantity-selector">
+                <label class="item-qty-label" for="item-qty"
+                  >Quantity:<input
+                    type="number"
+                    name="item-qty"
+                    class="item-qty"
+                    id="item-qty"
```
Comment (ApurvaSawant11) [inline Components/cart/cart.html:100]: For input, add starting `value=1` and also set `min=1`. It is taking negative values right now.

**Review by ApurvaSawant11 — APPROVED** [review]
Looking good.

**Review by partha8 — APPROVED** [review]
Have an add to wishlist button too, it's in the PRD.

File: Components/cart/cart.html (hunk 1 above)
Comment (ParthParolekar) [reply Components/cart/cart.html:100]: Done

**Comment by ParthParolekar** [comment]
> Have an add to wishlist button too, it's in the PRD.

Done! Thank you for reminding.

**Comment by ParthParolekar** [comment]
Thank you @partha8 and @ApurvaSawant11  for the reviews. Merging now

**Pull request merged**